libopae/src/reconf.c

```diff
@@ -133,6 +135,17 @@ static fpga_result validate_bitstream(fpga_handle handle,
 	}
 }
 
+static fpga_result read_port_error(char *errpath, char *path, uint64_t *error)
+{
+	snprintf_s_ss(errpath, sizeof(errpath), "%s/%s", path, PORT_SYSFS_ERRORS);
```

Comment:
Does this work? `sizeof(errpath)` is 8 (`errpath` is a pointer).

Reply:
fixed.